modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/TestPodScheduler.java

```diff
@@ -266,6 +277,38 @@ V1Pod createTestPod(String runName, String engineName, boolean isTraceEnabled) {
         return newPod;
     }
 
+    private List<V1Toleration> createNodeTolerations(String nodeTolerations) {
+        List<V1Toleration> tolerationsList = new ArrayList<>();
+
+        String[] tolerationStringSplit = nodeTolerations.split(",");
+
+        if(tolerationStringSplit.length > 0) {
+            for(int i = 0; i < tolerationStringSplit.length; i++){
+                String[] selection = tolerationStringSplit[i].split("=");
+
+                if (selection.length == 2) {
+                    String[] operatorAndEffect = selection[1].split(":");
```

Comment:
A comment showing the expected format of the provided tolerations list might make this easier to read in the future

Reply:
Does this need to be included in the code or is this covered by the documentation in the helm chart?